gradle/libs.versions.toml

```diff
@@ -50,8 +49,15 @@ photoview = "2.3.0"
 sparkbutton = "4.1.0"
 unified-push = "2.0.1"
 
+[plugins]
+android-application = { id = "com.android.application", version.ref = "agp" }
+android-library = { id = "com.android.library", version.ref = "agp" }
```

Comment:
Not used anywhere?

Reply:
We can remove `android-library`.

Reply:
10aa548081f9554edd7b27dabd0332dc0e1f6340